pyccel/codegen/printing/fcode.py

```diff
@@ -2897,16 +2938,16 @@ def indent_code(self, code):
                 continue
             level -= decrease[i]
 
-            padding = " "*(level*tabwidth + cont_padding)
+            padding = " "*(level*tabwidth)
 
             line = "%s%s" % (padding, line)
 
             new_code.append(line)
 
-            if continuation[i]:
-                cont_padding = 2*tabwidth
-            else:
-                cont_padding = 0
+#            if continuation[i]:
+#                cont_padding = 2*tabwidth
+#            else:
+#                cont_padding = 0
```

Comment:
Can this code be removed or do you think it will be useful later?

Reply:
It can be useful if we add the continuation lines in some print methods, or we allow the user to create the continuation lines by himself

Reply:
The same comment as before